### ISSUE TITLE
gnt-network has unclear error messages

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#494]
Originally reported of Google Code with ID 438.

```
# gnt-instance modify --net 0:ip=192.168.5.5 testvm6.local
Failure: prerequisites not met for this operation:
error type: wrong_state, error details:
Conflicting IP address found: '192.168.5.5' != 'cnet1'

This means that the ip address can't be used because it belongs to network "cnet1", but the specified NIC does not. The message should be clarified in this direction.


```
Originally added on 2013-04-24 08:19:11 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-04-26 12:39:32 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-04-29 08:36:48 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:49 +0000 UTC.

Changed State: Released
